Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl

```diff
@@ -9,4 +9,7 @@ ghost-role-information-mouse-description = A hungry and mischievous mouse.
 ghost-role-information-giant-spider-name = Giant spider
 ghost-role-information-giant-spider-description = Wreak havoc on the station's inhabitants!
 
-ghost-role-information-cognizine-description = Made conscious with the magic of cognizine.
\ No newline at end of file
+ghost-role-information-cognizine-description = Made conscious with the magic of cognizine.
+
+ghost-role-information-hamster-name = Hamster
```

Comment:
shouldnt hamlet have his own ghost role info?

Reply:
I musat have missed that, will take a look tomorrow

Reply:
hamlet now has his own unique ghost role description and name.